netaddr.go

```diff
@@ -90,10 +90,8 @@ func IPv4(a, b, c, d uint8) IP {
 	}
 }
 
-// IPv6Raw returns the IPv6 address given by the bytes in addr,
-// without an implicit Unmap call to unmap any v6-mapped IPv4
-// address.
-func IPv6Raw(addr [16]byte) IP {
+// IPFrom16 returns the IPv6 address given by the bytes in addr.
+func IPFrom16(addr [16]byte) IP {
```

Comment:
This violates the "new behavior, new name" adage. It's better to break existing callers than silently break them with a change in behavior.

If this will only return IPv6 now, maybe a better name is just `IPv6` to match https://godoc.org/inet.af/netaddr#IPv4.